app/src/main/java/com/sun/noteapp/utils/Helpers.kt

```diff
@@ -26,3 +29,31 @@ fun getListColor(): MutableList<Pair<Int?, String>> {
     return colors
 }
 
+fun getCurrentTime() = SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.getDefault()).format(Date())
+
+fun String.allIndexOf(word: String): List<Int> {
+    val lengthS = this.length
+    val lengthW = word.length
+    val result = ArrayList<Int>()
+    var i = 0
+    while (i < lengthS - lengthW) {
+        if (this.substring(i, i + lengthW).equals(word, true)) {
+            result.add(i)
+            i += lengthW - 1
+        }
+        i++
+    }
+    return result
+}
+
+fun getLabelsFromLabelDataString(labelDataStrings: List<String>): List<String> {
+    val result = mutableListOf<String>()
+    labelDataStrings.forEach {
+        val temp = it.split("_")
+        for (i in temp) {
```

Comment:
Kotlin có method distinct() cho Iterator.
```suggestion
        results = labels.distinct()
```